pkg/cri/server/sandbox_run_linux_test.go

```diff
@@ -28,7 +28,6 @@ import (
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 	runtime "k8s.io/cri-api/pkg/apis/runtime/v1"
-	v1 "k8s.io/cri-api/pkg/apis/runtime/v1"
```

Comment:
Q: with line 31 removed, do we still need to keep the `runtime` alias in line 30? (I remember we can also remove that alias right?)